DigitalOffice/Tests/ProjectServiceTests/Validators/EditProjectRequestValidatorTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using NUnit.Framework;
+using FluentValidation;
+using FluentValidation.TestHelper;
+using LT.DigitalOffice.ProjectService.Models;
+using LT.DigitalOffice.ProjectService.Validators;
+
+namespace LT.DigitalOffice.ProjectServiceUnitTests.Validators
+{
+    public class EditProjectRequestValidatorTests
+    {
+        private IValidator<EditProjectRequest> validator;
+        private EditProjectRequest editRequest;
+
+        [SetUp]
+        public void SetUp()
+        {
+            validator = new EditProjectValidator();
+
+            editRequest = new EditProjectRequest
+            {
+                Name = "Name",
+                Description = "Description",
+                IsActive = false,
+                DepartmentId = Guid.NewGuid()
+            };
+        }
+
+        [Test]
```

Comment:
Add test checks rules for empty Name, long Name

Reply:
Added